### DISCUSSION TITLE
Expose "Pose Score" for Custom Logic

### DISCUSSION THREAD

**Discussion by matttidridge** [endink/Mediapipe4u-plugin#128] [Q&A]
In the current execution, we can set a `MinPoseScoreThresh` where it looks like if a part of the body is not tracked, the character will default to the T pose. 

If the user is no longer being tracked or the tracking is poor for some reason, we would like the character to play a random idle animation instead. Is it possible to get access to the "Pose Score" or some kind of Bool that shows when the `Pose Score` is below the `MinPoseScoreThresh`?

**Comment by endink** [comment]
You can access origin mediapipe data in C++, HolisticComponent has some event, please see the .h file, XXXTrigger means an C++ native event.
